src/pages/FrontendMentor/YxZkCountdown/styles/YxZkCountdown.styled.ts

```diff
@@ -0,0 +1,112 @@
+import styled, { keyframes } from 'styled-components'
+
+export const BackgroundContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+  background-color: #1e1f29;
+  background-size: cover;
+  background-repeat: no-repeat;
+  position: relative;
+  overflow: hidden;
+  @media (max-width: 768px) {
+    width: 100%;
+  }
+`
+
+export const StarBackgroundContainer = styled.div`
+  display: flex;
+  position: relative;
+  height: 80vh;
+`
+export const StarBackground = styled.img`
+  width: 100vw;
+  display: flex;
+  position: relative;
+  background-position: top;
+  object-fit: cover;
+  @media (max-width: 768px) {
+  }
+`
+export const HillBackgroundContainer = styled.div`
+  background-size: cover;
+  height: 20vh;
+`
+export const HillBackground = styled.img`
+  width: 100vw;
+  height: 100%;
+  margin-top: auto;
+  margin-bottom: auto;
+  object-fit: cover;
+  @media (max-width: 768px) {
+  }
+`
+
+const FoldUp = keyframes`
+  from {
```

Comment:
consider removing FoldUp and FoldDown since u are not using them

Reply:
Fixed in commit e51bb11bf9a4a10dae44053fd14111a499882f29